snabble/Android-SDK
# Add animation for a succesfully checkout

### What's new?

A new animation is now shown as soon as the checkout is finished successfully. 

### How to test?

* deploy the branch
* check into a store -> buy a thing -> on success the animation should start
